mobile/lib/utils/pm.dart

```diff
@@ -13,65 +13,35 @@ List<Recommendation> getHealthRecommendations(
   final recommendations = <Recommendation>[];
```

Comment:
Thanks @BenjaminSsempala. 
Could we utilise this PR to rename Recommendation(s) to HealthTip(s).

Reply:
Thanks @NoahNsimbe 
I've made the changes.